{
  "msgDefault": "Error processing client request",
  "diagnoses": [
    {
      "message": "writeToStream error",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[writeToStream error | RuntimeException | com.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:462)]\nmess(2)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:74)]\nmess(3)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:86)]\nmess(4)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:186)]\nmess(5)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)]\nmess(6)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(7)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(8)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(9)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:462)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:74)]\nmess(2)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:86)]\nmess(3)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:186)]\nmess(4)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)]\nmess(5)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(6)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(7)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(8)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:74)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:86)]\nmess(2)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:186)]\nmess(3)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)]\nmess(4)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(5)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(6)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(7)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:186)]\nmess(2)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)]\nmess(3)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(4)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(5)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(6)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:186)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)]\nmess(2)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(3)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(4)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(5)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:57)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:167)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)]\nmess(2)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(3)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(4)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:151)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:53)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:167)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.RuntimeException: java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.RuntimeException: java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)]\nmess(2)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(3)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:719)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:134)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:53)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:167)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "message": "java.lang.NullPointerException",
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.RuntimeException]\nmess(1)=[java.lang.NullPointerException | RuntimeException | com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)]\nmess(2)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:233)\ncom.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJsonAddRecord:724)\ncom.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:672)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:134)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:53)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:167)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    },
    {
      "severity": "ERROR",
      "sdataCode": 500,
      "stackTrace": "\nclass=[java.lang.NullPointerException]\nmess(1)=[NullPointerException | com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)]\nstack=[com.sage.x3.bundle.shared.adapi.CAdapiVariable(getValueForJson:199)\ncom.sage.x3.bundle.shared.adapi.CAdapiDataSrcLine(toJson:222)\ncom.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJsonAddRecord:724)\ncom.sage.x3.bundle.shared.adapi.CAdapiDataSrc(toJson:672)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWin(toJson:134)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionWins(toJson:53)\ncom.sage.x3.bundle.internal.fusion.syndication.sap.CSDataPayloadFusionSAP(toJson:167)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:78)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataPayloadFusionResponse(toJson:1)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(toJSON:60)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(writeToStream:449)\ncom.sage.x3.shared.sdata.syndication.CSyndicationBase(dumpInString:86)\ncom.sage.x3.bundle.internal.fusion.syndication.CSDataEntryFusion(dumpInString:28)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.RequestSvc(execHandler:114)\ncom.sage.x3.bundle.internal.fusion.x3.handlerRequest.AbsRequestSvc(start:44)\ncom.sage.x3.bundle.internal.fusion.x3.FusionClient(execRequest:426)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(_execSession:250)\ncom.sage.x3.bundle.internal.fusion.sdata.adapter.FusionClientAdapter(execSession:365)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(putSession:741)\ncom.sage.x3.bundle.sdata.handler.CContractHandler(doPutTreatement:287)\ncom.sage.x3.shared.sdata.util.CSDataAbstractHandler(execute:87)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(doExec:128)\ncom.sage.x3.bundle.internal.sdata.CSDataContractServiceImpl(execRequest:152)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:188)\ncom.sage.x3.bundle.internal.sdata.server.CSDataSrvServiceImpl(execRequest:269)\ncom.sage.x3.bundle.internal.sdata.server.request.CHttpHandlerSDataRequest(doRequest:27)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequestDefault(doPut:30)\ncom.sage.x3.bundle.http.core.utils.CHandlerRequest(handle:52)\norg.apache.http.protocol.HttpService(doService:369)\norg.apache.http.protocol.HttpService(handleRequest:290)\ncom.sage.x3.bundle.internal.http.core.CWorkerThread(run:40)]\n"
    }
  ],
  "errMsg": "writeToStream error",
  "msgCode": 508,
  "errCode": "KILL"
}
